example/test/function_component_test.dart

```diff
@@ -5,19 +5,34 @@ import 'package:react/react.dart' as react;
 import 'package:react/react_dom.dart' as react_dom;
 import 'package:react/react_client.dart';
 
-var useStateTestFunctionComponent = react.registerFunctionComponent(UseStateTestComponent, displayName: 'useStateTest');
+var hookTestFunctionComponent = react.registerFunctionComponent(HookTestComponent, displayName: 'useStateTest');
 
-UseStateTestComponent(Map props) {
-  final count = useState(0);
+HookTestComponent(Map props) {
+  final count = useState(1);
+  final evenOdd = useState('even');
+
+  useEffect(() {
+    if (count.value % 2 == 0) {
+      print('count changed to ' + count.value.toString());
+      evenOdd.set('even');
+    } else {
+      print('count changed to ' + count.value.toString());
+      evenOdd.set('odd');
+    }
+    return () {
+      print('count is changing...');
+    };
+  }, [count.value]);
```

Comment:
```suggestion
  });
```
i understand why this was added, but it is not really necessary.